features/feature-queries.yml.dist

```diff
@@ -0,0 +1,77 @@
+# Generated from: feature-queries.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2015-09-30
+  baseline_high_date: 2018-03-30
+  support:
+    chrome: "28"
+    chrome_android: "28"
+    edge: "12"
+    firefox: "22"
+    firefox_android: "22"
+    safari: "9"
+    safari_ios: "9"
+compat_features:
+  # ⬇️ Same status as overall feature ⬇️
+  # baseline: high
+  # baseline_low_date: 2015-09-30
+  # baseline_high_date: 2018-03-30
+  # support:
+  #   chrome: "28"
+  #   chrome_android: "28"
+  #   edge: "12"
+  #   firefox: "22"
+  #   firefox_android: "22"
+  #   safari: "9"
+  #   safari_ios: "9"
+  - api.CSSSupportsRule
+  - css.at-rules.supports
+
+  # baseline: high
+  # baseline_low_date: 2021-04-26
+  # baseline_high_date: 2023-10-26
+  # support:
+  #   chrome: "83"
+  #   chrome_android: "83"
+  #   edge: "83"
+  #   firefox: "69"
+  #   firefox_android: "79"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - css.at-rules.supports.selector
+
+  # baseline: low
+  # baseline_low_date: 2023-09-18
+  # support:
+  #   chrome: "108"
+  #   chrome_android: "108"
+  #   edge: "108"
+  #   firefox: "106"
+  #   firefox_android: "106"
+  #   safari: "17"
+  #   safari_ios: "17"
+  - css.at-rules.supports.font-format
+  - css.at-rules.supports.font-tech
+
+  # baseline: low
+  # baseline_low_date: 2024-05-13
+  # support:
+  #   chrome: "121"
+  #   chrome_android: "121"
+  #   edge: "121"
+  #   firefox: "114"
+  #   firefox_android: "114"
+  #   safari: "17.5"
+  #   safari_ios: "17.5"
+  - api.CSSImportRule.supportsText
+
+  # baseline: false
+  # support:
+  #   chrome: "122"
+  #   chrome_android: "122"
+  #   edge: "122"
+  #   firefox: "115"
+  #   firefox_android: "115"
+  - css.at-rules.import.supports
```

Comment:
This and api.CSSImportRule.supportsText are the same feature, and any differences are either errors in BCD, or mishaps in shipping. The chromestatus entry is https://chromestatus.com/feature/5899007704694784.

@danielsakhapov, does it make sense that the API part shipped first in Chrome 121?

Reply:
Looking at https://github.com/mdn/browser-compat-data/pull/21603 and https://github.com/mdn/browser-compat-data/pull/22164 and https://issues.chromium.org/issues/40244647#comment15 it's also possible that it all shipped in 122?

(It's unclear to me whether the BCD collector correctly found `supportsText` in Chrome 121 beta, or if @Elchi3 added it because Chrome Status incorrectly said it was shipping.)

Reply:
Hmm, 121 is the correct version according to https://chromiumdash.appspot.com/commit/f9bbbf13060df98c24385bc83da7693d33f3650d

Reply:
@danielsakhapov helped me figure out what happened here. https://chromiumdash.appspot.com/commit/6ca8ac7e347b73c62609643164a444719f9d26f7 is the commit that actually shipped the feature, in Chrome 122.

The CSSOM bits were accidentally exposed in Chrome 121 and don't do anything. That's actually my fault, as the reviewer of that CL.

Reply:
For Safari, we'd need to check if the feature is enabled, or if they made the same mistake of exposing the API first.

Reply:
Hmm, there's is a mismatch in Firefox too...

@ddbeck I'd suggest just putting this subfeature to the side for now.

Reply:
OK, I've commented it out and left a note back to this discussion. We can get it to when we're in the long tail, I guess?